transactions/src/main/resources/org/jboss/as/txn/subsystem/LocalDescriptions.properties

```diff
@@ -41,7 +41,7 @@ transactions.enable-tsm-status=Set to 'true' to enable the transaction status ma
 transactions.default-timeout=The default timeout, in seconds, for a new transaction started by the transaction manager (see also the 'maximum-timeout' attribute).
 transactions.maximum-timeout=If the 'default-timeout' attribute is zero then this value is used for setting the maximum timeout value (in seconds) for newly started transactions.
 
-transactions.object-store-path=Denotes a path where the transaction manager object store should store data. By default the value is treated as relative to the path denoted by the 'relative-to' attribute. When the 'relative-to' attribute is undefined the value is considered an absolute path. This setting is valid when default or journal store is used. It's not used when the jdbc journal store is used.
+transactions.object-store-path=Denotes a path where the transaction manager object store should store data. By default, the value is treated as relative to the path denoted by the 'relative-to' attribute. When the 'relative-to' attribute is undefined the value is considered an absolute path. This setting is valid when default or journal store is used. It's not used when the jdbc journal store is used.
```

Comment:
`It's not used when the jdbc journal store is used.`: the journal store is not dependant on JDBC, it uses a filesystem.